google_symptoms/Makefile

```diff
@@ -13,7 +13,7 @@ install: venv
 install-ci: venv
 	. env/bin/activate; \
 	pip install wheel ; \
-	pip install ../_delphi_utils_python ;\
+	pip install '../_delphi_utils_python[dev]' ;\
```

Comment:
```suggestion
	pip install ../_delphi_utils_python ;\
```

Reply:
Has an interesting error; when I don't do the [dev] it's saying that covidcast doesn't have usi_api_key as a module???

`delphi_google_symptoms/date_utils.py:77:4: E1101: Module 'covidcast' has no 'use_api_key' member (no-member)`

https://github.com/cmu-delphi/covidcast-indicators/actions/runs/10285726380/job/28464696528